Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
Getting inspiration from existing recipes is good, directly using them (if possible) is even better :smile: 

Kind of like in the world of Invenio 1 and Invenio 2 where `invenio-kickstart` could install any overlay, either the  Invenio demo site, or the CDS overlay, or the INSPIRE overlay, all from the same source.  This seems more efficient than replicating installation recipes in each overlay to me...

For Invenio 3, the provisioning scripts are not yet that flexible, but it's a small step to get there, especially if we join forces across services.